### DISCUSSION TITLE
A blocking case with an ifStmt

### DISCUSSION THREAD

**Discussion by Sim4n6** [github/codeql#10537] [Q&A] [answered]
So I'm having a result of a query with false positives.
The false positives happen because the following code which is a barrier:

```python
// source
if node.evaluate():
    raise Error()
// sink
```

I would like to consider such a case. Eliminate with a isSanitizer() those considered True by an ifStatment, please?
Any help is a welcome? thank you

**Comment by edoardopirovano** [comment, marked as answer]
Greetings! Would you be able to share the query you're working on and a few details about what you're trying to do? I think with a little more context we'll be able to help give you some more effective suggestions.

**Reply by Sim4n6** [reply]
I promise next time I will do.
